live-examples/html-examples/css/article.css

```diff
@@ -0,0 +1,27 @@
+.forecast {
+    margin: 0;
+    padding: .3rem;
+    background-color: #eee;
+    font: 1rem "Fira Sans", sans-serif;
+}
+
+.forecast>h1, .day-forecast {
+    margin: .5rem;
+    padding: .3rem;
+    font-size: 1.2rem;
+}
+
+.day-forecast {
+    background: right/contain content-box border-box no-repeat url('/media/examples/rain.svg') white;
+}
+
+.day-forecast>h2, .day-forecast>p {
```

Comment:
Nit: Code Style: space between selectors, in this case the >. Also, each selector on its own line.